task5.css

```diff
@@ -0,0 +1,80 @@
+body {
+    background-color: #2f2d2d;
+}
+
+.cols {
+    width: 1000px;
+    height: 180px;
+    background-color: #ffffff;
+    border: 2px solid #ffffff;
+    border-radius: 15px;
+    padding-top: 8px;
+    padding-bottom: 8px;
+}
+
+.col {
+    float: left;
+    width: 33%;
+    text-align: center;
+}
+
+#center {
+    float: left;
+    width: 33%;
+    text-align: center;
+    border-left: 1px solid #bbb8b8;
+    border-right: 1px solid #bbb8b8;
+}
+
+img {
+    width: 100px;
+    height: 240px;
+}
+
+.col1,
+.col2,
+.col3 {
+    width: 100px;
+    height: 80px;
+    margin-left: -50px;
+    overflow: hidden;
+    position: relative;
+    left: 50%;
+}
+
+.col2 img {
+    position: absolute;
+    top: -81px;
+    left: 0;
+    right: 0;
+}
+
+.col3 img {
+    position: absolute;
+    top: -161px;
+    left: 0;
+    right: 0;
+}
+
+h3 {
+    color: #fd531f;
+    margin: 10px;
+}
+
+p {
+    font-size: 16px;
+    font-family: Arial;
+    clear: both;
+    margin: 0;
+}
+
+button {
```

Comment:
button looks not good 
![image](https://user-images.githubusercontent.com/40792121/78402356-7ce01700-760b-11ea-8393-85bf76c374b5.png)